common/changes/@microsoft/gulp-core-build/replace_depricated_gulp_util_2018-10-16-10-10.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/gulp-core-build",
+      "comment": "Replace deprecated dependency gulp-util",
+      "type": "minor"
```

Comment:
>minor [](start = 15, length = 5)

Are any of these types exposed via the API?  If not, we could make it a "patch" update